benchmarks/simulator.py

```diff
@@ -0,0 +1,37 @@
+from pysb.examples import earm_1_0
+from pysb.simulator import ScipyOdeSimulator, CupSodaSimulator
+import numpy as np
+import timeit
+
+
+class Earm10ODESuite(object):
+    def setup(self):
+        self.nsims = 100
+        self.timer = timeit.default_timer
+        self.model = earm_1_0.model
+        self.parameter_set = np.ones((self.nsims, len(self.model.parameters)))
```

Comment:
One of those "what the hell was I thinking" moments. Your solution is much better, will use it. Thanks.